src/corrgi/dask.py

```diff
@@ -29,9 +32,11 @@ def compute_autocorrelation_counts(catalog: Catalog, random: Catalog, params: Mu
     # Calculate the angular separation bins
     bins, _ = gundam.makebins(params.nsept, params.septmin, params.dsept, params.logsept)
     params_dd, params_rr = generate_dd_rr_params(params)
+    # Create correlation with bins and params
+    correlation = corr_type(bins, params)
     # Generate the histograms with counts for each catalog
-    counts_dd = perform_auto_counts(catalog, bins, params_dd)
-    counts_rr = perform_auto_counts(random, bins, params_rr)
+    counts_dd = perform_auto_counts(catalog, correlation)
+    counts_rr = perform_auto_counts(random, correlation)
```

Comment:
is it valid to use the same params for these now? Do you not need two correlations with params_dd and params_rr?

Reply:
Since we're calling the Fortran routines directly for now we just need "nsept, septmin, dsept, and logsept" to be able to calculate the bins. I might need to move the parameter creation later on to each of the "angular" / "redshift" / "projected" if they turn out to vary according to the type of correlation, but didn't want to jump on that just yet.